### PR TITLE
Refactor yaml format

### LINKED ISSUE #1: YAML Format
The current YAML file includes full sentences as keys and redundant data. Shorter keys and deduplication would help to prevent cluttering the file. 

Suggestion: two yaml files
`cs-groups.yaml`
```yaml
- name: Chair of Distributed and Networked Systems
  handle: NETD
  website: https://netd.cs.tu-dresden.de
- full: Chair for Compiler Construction
  handle: CCC
  website: https://cfaed.tu-dresden.de/ccc-about
```

`oss-projects.yaml`
```yaml
- name: RIOT
  description: RIOT is the friendly Operating System for the Internet of Things.
  website: https://riot-os.org
  repository: https://github.com/RIOT-OS/RIOT
  license: LGPL
  chair: [ "NETD" ]
  founder: true
  maintainer: true
  contributor: true
  involved: true
  logo: RIOT-logo.png
```

### PR DESCRIPTION
**Pull request by PaulPickhardt**
Splits project yaml in oss-projects and cs-groups and refactors the yaml keys.

Closes #1

### CODE REVIEW

**Comment by PaulPickhardt** [comment]
I'm done with refactoring the Yaml files as supposed in issue #1. I changed `chair` to `groups` in `oss-projects.yaml`. 

Let me know what you think @waehlisch @adeveloper-wq

File: README-template.md [hunk 1]
```diff
@@ -1,44 +1,57 @@
 # TU Dresden - Computer Science: Open Source Projects
```
Comment (waehlisch) [inline README-template.md]: Open Source Projects of the Faculty of Computer Science at TU Dresden

File: README-template.md [hunk 2]
```diff
@@ -1,44 +1,57 @@
...
 
-This repository contains functionality to generate a HTML and markdown site of all projects entered in `projects.yaml`.
+This repository contains functionality to generate a HTML and markdown site of all projects entered in `oss-projects.yaml` and computer science groups in `cs-groups.yaml`.
```
Comment (waehlisch) [inline README-template.md]: This repository contains scripts and data to generate HTML and markdown files presenting open source software projects of the [Faculty of Computer Science](https://cs.tu-dresden.de) at [TU Dresden](https://www.tu-dresden.de). The sources of information are `oss-projects.yaml`, which describes the projects, and `cs-groups.yaml`, which describes the research groups involved.

File: README-template.md [hunk 3]
```diff
@@ -1,44 +1,57 @@
...
+  maintainer:
+  contributor:
+  involved:
+  logo:
 ```
 
-For the image, upload the image to the `images` folder and enter the relative URL to the image in the `images` folder.
+For the logo, upload the logo to the `images/projects` folder and enter the relative URL to the logo in the `images/projects` folder.
 
+For the groups add the handle of your computer science group to the list.
```
Comment (waehlisch) [inline README-template.md]: For the groups, add the handle of your research group to the list.

File: README-template.md [hunk 4]
```diff
@@ -1,44 +1,57 @@
...
+
+If your computer science group does not exist yet, append this template to `cs-groups.yaml` and fill in your data:
```
Comment (waehlisch) [inline README-template.md]: If your research group [...]

**Review by waehlisch — CHANGES_REQUESTED** [review]
in addition to what we just talked about.

File: README.md [hunk 5]
```diff
@@ -1,47 +1,60 @@
...
+For the groups add the handle of your computer science group to the list.
+
+If your computer science group does not exist yet, append this template to `cs-groups.yaml` and fill in your data:
+```yaml
+- name: 
+  handle: 
+  website: 
+```
 
 # Projects
```
Comment (waehlisch) [inline README.md:59]: Paragraph below: The following list presents open source software projects in which members of the Faculty of Computer Science at TU Dresden are involved.

**Comment by PaulPickhardt** [comment]
The description is now optional and I added your suggestions to the readme @waehlisch

**Review by waehlisch — APPROVED** [review]

**Pull request merged**